botbot_plugins/tests/test_vote.py

```diff
@@ -0,0 +1,247 @@
+# -*- coding: utf-8 -*-
+import pytest
+from botbot_plugins.base import DummyApp
+from botbot_plugins.plugins import vote
+
+
+@pytest.fixture
+def app():
+    return DummyApp(test_plugin=vote.Plugin(), command_prefix="!")
+
+def test_no_concurrent_voting(app):
+    assert app.respond("!startvote") == ["Voting has started."]
+    assert app.respond("!startvote") == [u"repl_user: There’s already a vote going on. Use the “endvote” command to end it before starting a new one."]
+    app.respond("!endvote")
+
+def test_no_vote_running(app):
+    assert app.respond("+1") == []
+    assert app.respond("+something") == []
+    assert app.respond("!vote +something") == [u"No vote has been started. Use the “startvote” command to do so."]
+    assert app.respond("!countvotes") == [u"No vote has been started. Use the “startvote” command to do so."]
+    assert app.respond("!abstain") == [u"No vote has been started. Use the “startvote” command to do so."]
+    assert app.respond("!cancelvotes") == [u"No vote has been started. Use the “startvote” command to do so."]
+    assert app.respond("!endvote") == [u"No vote has been started. Use the “startvote” command to do so."]
+
+def test_boolean_voting(app):
```

Comment:
I can see lots of duplication in the test suite. Use unittest module and move common code to test setup and cleanup